### ISSUE TITLE
Support stopped on certain programs/websites

### ISSUE THREAD

**Issue by RandomLion** [FrostCo/AdvancedProfanityFilter#583]
The extension is no longer working on Google sites on Microsoft Edge, and also doesn't work on Source Editor on Fandom websites on any browser.

**Comment by MaryAtlas** [comment]
I think author has gone away? I posted something similar a few days ago and there was no response.

**Comment by richardfrost** [comment]
@MaryAtlas, I'm still here, just been really busy with life and haven't had as much time to invest here. I do plan on getting to everything eventually though! Thanks for your patience and support.

**Comment by richardfrost** [comment]
@RandomLion, can you confirm which store you got the extension from for Microsoft Edge?

**Comment by RandomLion** [comment]
@richardfrost I got the extension from the Chrome Web Store. Also, Google website support for the Advanced Profanity Filter on the Google Chrome browser just stopped. Please fix it.

**Comment by richardfrost** [comment]
I am not able to see the issue that you are describing @RandomLion, so lets see if we can get to the bottom of it.

If you open the extension's Popup, (by clicking on the icon in the browser), do you see a message at the bottom about the filter being disabled like this?

![image](https://github.com/user-attachments/assets/150f5268-6c00-410a-8722-864ce94b05e8)

If so, what is the message?

To help with this problem more, could you choose one of the following options to help get to the bottom of this?

1. Share a copy of your config so I can more closely replicate your situation? To do that, open up the extension's Options page > Config tab and click the "Export" button. That will save a `.json` file to your computer, and then you can either attach it here, or send an email to support @ frostco.dev (without the spaces).
2. Make a backup of your config (open up the extension's Options page > Config tab and click the "Export" button). Once you have the file, click the "Restore Defaults" on the same page. Once it has been reset, feel free to add a test word or two to your list (something simple like `apple`, or `the` work well, since they shouldn't be offensive but let you test the effectiveness of the filter. If that works, it means that there must be one of your settings preventing the filter from working the way you'd like. If that is the case you can try to track it down. I'll list a few of the most common ones below:
    - Audio tab > Only mute audio: This option prevents the filter from running on regular text everywhere. If a site isn't a supported audio site, the filter will be disabled.
    - Settings tab > Filter Method > Off: Disabled the filter, but still lets it count and summarize which words would have been filtered
    - Domains tab > Minimal mode: This domain mode will only run the filter on sites that you specifically tell it to, and will be disabled on all others
    - Domains tab > Domains list: This list contains all the domains that you have changed settings for. If you have a domain listed here, you can choose to disable the filter on that page, which may not be intended. Check to see if there are any domains listed that you may want to adjust settings for.

**Comment by richardfrost** [comment]
@RandomLion Are you still having the issue that you described?

**Comment by RandomLion** [comment]
Not at all. But Advanced Profanity Filter still no longer works in Fandom's source editor, when it did just fine several months ago.
